README.md

```diff
@@ -172,6 +172,12 @@ Then fill in the generated file with logic, fixtures, etc. You can then apply th
 
 Running the Development Server
 ------------------------------
+You will need to have the Node.js package manager npm installed. You can install it by running:
```

Comment:
This isn't a totally accurate description. We're not installing the Node.js package manager (npm), we are in fact using npm to install other dependencies that are required for One and Done.